app/lib/find_constraint.rb

```diff
@@ -2,6 +2,6 @@
 
 class FindConstraint
   def matches?(request)
-    Settings.find_temp_url&.include?(request.host)
+    Settings.find_temp_url&.include?(request.host) || request.host.include?("find2-pr")
```

Comment:
Hmmm on second thought, maybe this is better, as we don't really know the review env url and we don't care enough for the test env.

```suggestion
    Settings.find_temp_url&.include?(request.host) || Rails.env.review? || Rails.env.test?
```

Then we can remove `StubbedFindConstraint` altogether and the noise it created.

Reply:
I don't think this makes sense because we need to be explicit to only activate the constraint depending on the URL (which will be different for find and publish review apps) otherwise some of the routes will conflict if both constraints are activated without checking it. Same re: the test, the stub gives more granular control